selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -692,11 +692,22 @@ public void typeTextIntoEditor(String text, int line) {
   }
 
   /**
-   * Sets cursor to specified {@code positionLine}.
+   * Sets cursor to specified {@code positionLine} and checks the result
    *
    * @param positionLine line's number where cursor should be placed
    */
   public void setCursorToLine(int positionLine) {
+    goToLine(positionLine);
+    waitActive();
+    expectedNumberOfActiveLine(positionLine);
+  }
+
+  /**
+   * Sets cursor to specified {@code positionLine}
+   *
+   * @param positionLine line's number where cursor should be placed
+   */
+  public void goToLine(int positionLine) {
```

Comment:
Name `goToLine()` has the same meaning as `setCursorToLine()`, IMHO.
I would rename `goToLine() ` to `moveCursorToLine()'.

Reply:
OK, I don't mind